rasa/nlu/extractors/extractor.py

```diff
@@ -254,6 +262,50 @@ def _update_confidence_values(
                 confidences[ENTITY_ATTRIBUTE_GROUP][idx],
             )
 
+    @staticmethod
+    def _check_is_single_entity(
```

Comment:
What about a case like `We need to get more of tofu cauliflower avocado`? The user was too lazy to input the commas. No matter what we would always extract `tofu cauliflower avocado` as a single entity, correct? Do you see a way how we could avoid that or is that something we don't want to support.

Reply:
Thats a tricky case and I have a feeling that trying to handle this might introduce more complexity and problems than it solves. 

We might want to re-visit this issue though if it turns out to be an annoying & recurring thing for users.

Reply:
yeah, not sure if we can actually support all the different scenarios. Let's merge as it is and then we can see what users think about it. What do you think?

Reply:
💯 - that sounds like the right plan!